core/src/main/scala/caliban/execution/Fragment.scala

```diff
@@ -7,7 +7,7 @@ case class Fragment(name: Option[String], directives: List[Directive]) {}
 
 object Fragment {
   object IsDeferred {
-    def unapply(fragment: Fragment): Option[Option[String]] =
+    def unapply(fragment: Fragment): Option[(Option[String])] =
```

Comment:
Out of curiosity, is there a reason for this change or is it a dev leftover?

Reply:
Artifact of dev, I think I was playing around with adding more context and compiler apparently doesn't complain about a tuple of one